recipes/exiv2/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+import os
+from conans import ConanFile, CMake, tools
+
+required_conan_version = ">=1.33.0"
+
+
+class Exiv2Conan(ConanFile):
+    name = "exiv2"
+    description = "Exiv2 is a C++ library and a command-line utility " \
+                  "to read, write, delete and modify Exif, IPTC, XMP and ICC image metadata."
+    license = "GPL-2.0"
+    topics = ("conan", "image", "exif", "xmp")
+    homepage = "https://www.exiv2.org"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake", "cmake_find_package"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
```

Comment:
Should we expose all the options of the library? https://github.com/Exiv2/exiv2/blob/v0.27.4/CMakeLists.txt#L15-L39
Some, like `ENABLE_BMFF` are missing here.

Reply:
Only the options relevant to users should be added.
`EXIV2_ENABLE_BMFF` might be interesting to them.
Other options might also be interesting.

But it's not required to make available all options.
The recipe can always be refined in a later pr.

Remember, it's easier to add options then it is to remove options.

Reply:
Actually, I had checked other options like BMFF or LENSDATA. They only enable some functionality in the library, do not bring other dependencies and they are on by default. I think if they are enabled by default, there is no harm to keep them enabled too.

XMP is a different kind of option. It produces another binary so I added a conanfile option.

NLS is off by default, brings a dependency so I added an explicit CMake option.